app/models/UserAnswers.scala

```diff
@@ -49,7 +50,8 @@ object ReadOnlyUserAnswers {
 final case class UserAnswers(draftId: String,
                              data: JsObject = Json.obj(),
                              internalAuthId :String,
-                             override val is5mldEnabled: Boolean = false) extends ReadableUserAnswers {
+                             override val is5mldEnabled: Boolean = false,
+                             override val isTaxable: Boolean = true) extends ReadableUserAnswers {
```

Comment:
Does this work include anything relating to non-taxable? If so I think we can leave this change out for now

Reply:
It is Needed, some of the questions only get asked if its taxable

Reply:
But until the Non-Taxable journey is implemented we assume it to be taxable anyway. I think you can just leave this code to be done by whoever picks up [TRUS-3503](https://jira.tools.tax.service.gov.uk/browse/TRUS-3503).

Reply:
Ok, fair enough. Could be worth leaving a TODO in IndexController for whoever picks up one of the non-taxable tickets that isTaxable will need to be determined and set in user answers